lib/policies/jwt/jwt.js

```diff
@@ -21,7 +21,7 @@ module.exports = function (params) {
     }
 
     if (!params.checkCredentialExistence) {
-      return done(null, { id: 'anonymous' });
+      return done(null, jwtPayload);
```

Comment:
When creating a pull request, please start from the master branch, not from another one.